test/setup-fedora-govm.sh

```diff
@@ -52,17 +52,17 @@ gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg https://packages.cl
 EOF
 
     # For the sake of reproducibility, use fixed versions.
-    # List generated with:
-    # for v in 1.13 1.14 1.15 1.16 1.17 1.18; do for i in kubelet kubeadm kubectl; do echo "$i-$(sudo dnf --showduplicates list kubelet | grep " $v"  | sed -e 's/.* \([0-9]*\.[0-9]*\.[0-9]*[^ ]*\).*/\1/' | sort -u -n -k 2.6 | tail -n 1)"; done; done
-    # Note that -n and -k 2.6 are needed to handle minor versions > 9, and -k 2.6 works only if k8s major versions have form X.XX
+    # List of latest packages can be generated on Fedora with:
+    # for v in $(seq 13 20); do for i in kubelet kubeadm kubectl; do echo "$i-$(sudo dnf --showduplicates list kubelet | grep " 1.$v"  | sed -e 's/.* \([0-9]*\.[0-9]*\.[0-9]*[^ ]*\).*/\1/' | sort -u -n -k 1.6 | tail -n 1)"; done; done
```

Comment:
I find the `$(seq 13 20)` less readable. Previously it was obvious at a glance that these are Kubernetes version numbers, now not so much. It's not even that much shorter.

Reply:
If that list does not bother anybody, I can put it back, you are right code does not get much smaller. Speaking about list length BTW, do we still want to keep older versions here, even if we start testing with 1.17 ? If we only want older ones (13..16) for historical reference, older versions can as well remain in git history?